packages/devtools-config/src/feature.js

```diff
@@ -50,14 +58,26 @@ function getConfig() {
   return config;
 }
 
+function updateLocalConfig(relativePath) {
+  const localConfigPath = path.resolve(relativePath, "../configs/local.json");
+  try {
+    fs.writeFileSync(localConfigPath, JSON.stringify(config, null, "  "));
```

Comment:
nit: could you replace `"  "` with `2` as the third parameter to `JSON.stringify`?